conda-recipe/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
 $PYTHON setup.py install
-mkdir -p $SP_DIR/notebook/static
-cp -r static/pivot-ajax $SP_DIR/notebook/static
-cp -r static/pivottable $SP_DIR/notebook/static
+mkdir -p $PREFIX/share/jupyter/nbextensions/
+cp -r pivottablejs/static/pivot-ajax $PREFIX/share/jupyter/nbextensions/
```

Comment:
why pivottablejs/static here? is this building locally if you try to run conda build?